docs/team/wpinrui.md

```diff
@@ -4,14 +4,40 @@ title: Ivan's Project Portfolio Page
 ---
 
 ### Project: TutorAid
-
+<!-- PPP format and structure adapted from
+https://github.com/samuelfangjw/tp/blob/master/docs/team/samuelfangjw.md
+-->
 TutorAid is a desktop application used for helping tech-savvy private tutors, who have busy schedules / many students, to keep track of each students' details. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
 
 Given below are my contributions to the project.
-
+* **Card-like UI elements**:
+  * Added the ability to view lessons in a panel in TutorAid [(PR #106)](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/106)
+  * Justification: This feature is important for user functionality as having a separate panel for lessons allows them to manage lessons and students simultaneously.
+* **`list -a` command to show all fields while viewing a list of students and lessons [(PR #87)](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/87)**
+  * `list` command alone hides fields [(PR #70)](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/70)
+  * Justification: Having a list command that can hide details while listing students and lessons allows users to see more entries on a single screen, while having a list command that can show all details can allow users to look at fields of each of their students / lessons. 
+* **Edit [student](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/86) / [lesson](https://github.com/AY2122S1-CS2103T-W16-3/tp/pull/118) commands**
+  * Justification: Being able to edit is a feature that makes it a lot more convenient for users to change fields for students / lessons. Without this feature, they would have to delete the original entry and create a new one which is a lot of trouble, especially if only a single field has changed.
```

Comment:
A very minor nit-pick 😅 You can consider shifting the comma to '`...create a new one, which is a lot of trouble especially...`' so that the sentence flows better